pkg/cloud/scope/machinepool.go

```diff
@@ -43,11 +43,14 @@ import (
 	"sigs.k8s.io/cluster-api/util/patch"
 )
 
+const ReplicasManagedByAutoscalerAnnotation = "cluster.x-k8s.io/replicas-managed-by-autoscaler"
+
 // MachinePoolScope defines a scope defined around a machine and its cluster.
 type MachinePoolScope struct {
 	logr.Logger
 	client.Client
-	patchHelper *patch.Helper
+	patchHelper            *patch.Helper
+	MachinePoolPatchHelper *patch.Helper
```

Comment:
Ideally we'd want to not export this and provide a `PatchMachinePool` function on the scope. But i'm not so concerned about this

Reply:
I wondered if I should do it that way or provide a specific function on the scope. I don't mind changing this.

Reply:
adjusted :+1: